terraform/environments/mlra/alb.tf

```diff
@@ -58,7 +55,7 @@ module "alb" {
 
   listener_protocol = "HTTPS"
   listener_port     = 443
-  alb_ssl_policy    = "ELBSecurityPolicy-TLS-1-2-2017-01" # TODO This enforces TLSv1.2. For general, use ELBSecurityPolicy-2016-08 instead 
+  alb_ssl_policy    = "ELBSecurityPolicy-TLS13-1-2-2021-06"
```

Comment:
Are you sure uplifting this is not going to break anything? It appears from the comment we're enforcing TLSv1.2 - most probably for some good reason.

Reply:
Fair point, have reverted, to be revisited as part of LASB-3390.